public/static/js/projects.js

```diff
@@ -550,6 +554,16 @@ mciModule.controller('ProjectCtrl', function ($scope, $window, $http, $location,
     delete $scope.patch_alias
   };
 
+  $scope.addWorkstationCommand = function() {
+      if (!$scope.settingsFormData.workstation_config) {
+          $scope.settingsFormData.workstation_config = [];
+      }
+      if ($scope.validWorkstationCommand($scope.workstation_command)) {
```

Comment:
This should show an error if invalid. Though if there's not actually a way to get here if it's invalid, you don't need to check at all